appveyor-windows.yml

```diff
@@ -119,10 +119,7 @@ install:
   # Echo final Path
   - "echo %PATH%"
 
-  - ps: "
-      If (Test-Path env:BY_CANARY){
-      echo Logging in Public ECR; aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws;
-      }"
+  - "IF \"%BY_CANARY%\"==\"\" ECHO Logging in Public ECR; aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws;"
```

Comment:
for some unknown reason command `aws ecr-public get-login-password --region us-east-1` in Power shell returns extra line, and this cause the docker login command to fail. I had to change this call to use normal cmd

Reply:
if this was transient, let's aim to revert it.